webapp/src/main/java/com/box/l10n/mojito/service/leveraging/LeveragerByContent.java

```diff
@@ -40,6 +42,23 @@ public List<TextUnitDTO> getLeveragingMatches(TMTextUnit tmTextUnit, Long source
         return textUnitSearcher.search(textUnitSearcherParameters);
     }
 
+    public List<TextUnitDTO> getLeveragingMatchesForRepository(TMTextUnit tmTextUnit, Long sourceTmId, ArrayList<Long> repositoryIds, ArrayList<String> repositoryNames) {
```

Comment:
nit: this should be in another class since it is not used by this leverager. probably fine in the caller class?